apis/config/v1alpha1/types_landscaper_config.go

```diff
@@ -67,3 +69,13 @@ type OCICacheConfiguration struct {
 	// +optional
 	Path string `json:"path"`
 }
+
+// CrdManagementConfiguration contains the configuration of the CRD management
+type CrdManagementConfiguration struct {
+	// DeployCustomResourceDefinitions specifies if CRDs should be deployed
+	DeployCustomResourceDefinitions bool `json:"deployCrd"`
```

Comment:
Should we maybe default it to `true`

Reply:
The configuration of the CRD management was meant to be mandatory but yes, it makes more sense to keep it optional with `deployCrd: true` as default. Fixed!